calendar.js

```diff
@@ -97,30 +151,34 @@ function createTable() {
         td = document.createElement('td');
         td.innerText = i + ":00";
         tr.appendChild(td);
-        thElements.forEach(function(day, j) {
+        thElements.forEach(createTableCells);
+    }
+
+    function createTableCells(day, j) {
```

Comment:
please move this inner function to the top of the outer function (it's a good idea to group inner functions together at the top)